package.json

```
@@ -56,12 +56,22 @@
     "mocha": "7.1.2",
     "rollup": "2.9.1",
     "rollup-plugin-terser": "5.3.0",
+    "semantic-release": "^17.0.8",
     "ts-node": "8.10.1",
     "typescript": "3.9.2",
     "utf-8-validate": "5.0.2"
   },
   "mocha": {
     "spec": "src/**/tests/*.ts",
     "require": "env/setup-test-env.js"
-  }
+  },
+  "release": {
+		"branches": [
+			{
+				"name": "stable",
+				"channel": "latest"
+			}
+		],
+		"tagFormat": "v${version}"
```

```suggestion
		]
```
this is the [default](https://semantic-release.gitbook.io/semantic-release/usage/configuration#tagformat) so should automatically be there. Semrel is finicky though, so no worries if you ignore this :)